crates/store/src/db/sql.rs

```diff
@@ -483,17 +486,19 @@ pub fn select_notes_since_block_by_tag_and_sender(
         let note_type = row.get::<_, u8>(4)?;
         let sender = column_value_as_u64(row, 5)?;
         let tag: u32 = row.get(6)?;
-        let merkle_path_data = row.get_ref(7)?.as_blob()?;
+        let aux: u64 = row.get(7)?;
+        let aux = aux.try_into().map_err(|err| DatabaseError::DeserializationError(DeserializationError::InvalidValue(err)))?;
+        let merkle_path_data = row.get_ref(8)?.as_blob()?;
         let merkle_path = MerklePath::read_from_bytes(merkle_path_data)?;
-        let details_data = row.get_ref(8)?.as_blob_or_null()?;
+        let details_data = row.get_ref(9)?.as_blob_or_null()?;
         let details = details_data.map(<Vec<u8>>::read_from_bytes).transpose()?;
 
         // TODO: Properly retrieve aux
```

Comment:
We should be good to remove this `TODO`!

Reply:
Also, these same changes need to be applied to the other note queries

Reply:
done! I removed the TODO and applied the same change to `select_note` and `select_note_by_id`